cmd/thanos/receive.go

```diff
@@ -853,6 +860,18 @@ func (rc *receiveConfig) registerFlag(cmd extkingpin.FlagClause) {
 		Default("false").Hidden().BoolVar(&rc.allowOutOfOrderUpload)
 
 	rc.reqLogConfig = extkingpin.RegisterRequestLoggingFlags(cmd)
+
+	cmd.Flag("receive.request-limits.max-series",
+		"The maximum amount of series accepted in remote write requests.").
+		Default("0").IntVar(&rc.writeSeriesLimit)
+
+	cmd.Flag("receive.request-limits.max-samples",
+		"The maximum amount of samples accepted in remote write requests.").
+		Default("0").IntVar(&rc.writeSamplesLimit)
+
+	cmd.Flag("receive.request-limits.max-size-bytes",
+		"The maximum size (in bytes) of remote write requests.").
+		Default("0").IntVar(&rc.writeRequestSizeLimit)
```

Comment:
Do we want to support limits on the tenant level?

Reply:
Yup, this is something that I plan to add as a follow up. I am also talking with @saswatamcode, because his active series work will also need different per-tenant limits. We will propose that all the limits should be configured using a yaml file to make it simpler.

Reply:
So I am curious if we are going the per-tenant limit way, do we still want to keep those flags in this PR? Is it working as the default value if no limit is explicitly configured in the config file?

Reply:
Good question, @yeya24. I would prefer to have everything in the same place, the file. But defining the default value quickly through CLI args seem convenient too. I have no strong opinion on this. 

We can maybe support CLI and file for default limit and document that one of them has priority over the other.